style.css

```diff
@@ -591,7 +591,7 @@ main {
     justify-content: space-between;
     align-items: center;
     width: 90%;
-    height: 5vh;
+    height: 10vh;
     padding-bottom: 1vh;
     border-bottom: 1px solid rgba(0, 0, 0, 0.242);
     margin-left: 7%;
```

Comment:
- [ ] For the background image, it is getting in the message text area please match the original design 

- Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/220395596-0426e231-0f63-4268-815e-7b6663c14d33.png)

- Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/220395926-485125e0-b319-471e-8390-2fde6e2bff6c.png)